src/main/scala/org/camunda/feel/syntaxtree/Val.scala

```diff
@@ -186,6 +179,34 @@ case class ValYearMonthDuration(value: YearMonthDuration) extends Val {
 }
 
 case class ValDayTimeDuration(value: DayTimeDuration) extends Val {
+  override def toString: String = {
+    val millis = value.toMillis
+    val positive = millis > 0
+    val timeDuration = if (! positive) {
+      java.time.Duration.ofMillis(millis).negated()
+    } else {
+      java.time.Duration.ofMillis(millis)
+    }
+    val day = if(timeDuration.toDays != 0){
+      BigInteger.valueOf(timeDuration.toDays)
+    } else { null }
+    val hour = if(timeDuration.toHours % 24 != 0){
+      BigInteger.valueOf(timeDuration.toHours % 24)
+    } else { null }
+    val minute = if(timeDuration.toMinutes % 60 != 0){
+      BigInteger.valueOf(timeDuration.toMinutes % 60)
+    } else { null }
+    val seconds = if(timeDuration.getSeconds % 60 != 0){
+      BigInteger.valueOf(timeDuration.getSeconds % 60)
+    } else { null }
+    val dur = DatatypeFactory.newInstance().newDurationDayTime(
+      positive,
+      day,
+      hour,
+      minute,
+      seconds)
+    dur.toString
```

Comment:
:x: We should not use the class from Java's XML package. 

I would like to avoid confusion about the different date-time classes in Java and use the new types in `java.time.*` only.

Instead, we could build the string ourselves using string concatenation or string inter-polarisation.